src/DataFixtures/DemoFixtures.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\SnowCategory;
+use App\Entity\SnowComment;
+use App\Entity\SnowFigure;
+use App\Entity\SnowImage;
+use App\Entity\SnowUser;
+use App\Entity\SnowVideo;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+
+class DemoFixtures extends Fixture
+{
+    protected $userPasswordHasher;
+
+    public function __construct(UserPasswordHasherInterface $userPasswordHasher)
+    {
+        $this->userPasswordHasher = $userPasswordHasher;
+    }
+
+    public function load(ObjectManager $manager)
```

Comment:
Utilise des fonctions pour séparer ta fonction load.
`$this->loadCategories()`
`$this->loadFigures()`
`$this->loadUser()`
...

Reply:
Ca ne se fait plus depuis l'âge de pierre de faire une fonction de 300 lignes .... :D

Reply:
Tu pourrais même séparer celà en plusieurs class : https://symfony.com/bundles/DoctrineFixturesBundle/current/index.html#loading-fixtures

Reply:
Done